qualtrics_link/util.py

```diff
@@ -115,6 +115,40 @@
     'GSE-PPE [no longer used]': 'DV_0vsxWeIjXJWeS21'
 }
 
+# Maps the user type to its equivalent Qualtrics ID
+USER_TYPE_MAPPING = {
+    'employee': 'UT_egutew4nqz71QgI',
+    'student': 'UT_787UadC574xhxgU',
+    'brand administrator': 'UT_BRANDADMIN'
+}
+
+
+# Choice tuple used in the Qualtrics internal admin form
+DIVISION_CHOICES = (
+    ('API Div', 'API Div'),
+    ('Berkman', 'Berkman'),
+    ('Central Administration', 'Central Administration'),
+    ('EXT', 'EXT'),
+    ('FAS', 'FAS'),
+    ('GSE', 'GSE'),
+    ('GSE-PPE [no longer used]', 'GSE-PPE [no longer used]'),
```

Comment:
Nitpick.  Use similar parenthesis - either round or square . (Line 134 vs line 136).
Also, if the  GSE-PPE is no longer used, should it be displayed in the dropdown?

Reply:
That is the name as it displays in Qualtrics, same for the one on 136

Reply:
We should  check with PO if we should be displaying  unused divisions and let users be assigned to them? If we  do, then lets switch it to round brackets to keep it consistent.

Reply:
Talked offline about possibly reviewing this and cleaning it up  as part of a tech debt that's out there